code/script.js

```diff
@@ -1,47 +1,150 @@
 // All the DOM selectors stored as short variables
 const chat = document.getElementById('chat')
+const form = document.getElementById('name-form')
+const inputText = document.getElementById('name-input')
+const inputWrapper = document.getElementById('input-wrapper')
+const messageAudio = new Audio ("./assets/button_click.mp3");
 
 // Global variables, if you need any, declared here
 
 // Functions declared here
 
 // This function will add a chat bubble in the correct place based on who the sender is
 const showMessage = (message, sender) => {
+  console.log ('hi there')
   if (sender === 'user') {
     chat.innerHTML += `
       <section class="user-msg">
         <div class="bubble user-bubble">
           <p>${message}</p>
         </div>
-        <img src="assets/user.png" alt="User" />  
+        <img src="assets/girl.png" alt="User" />  
       </section>
     `
   } else if (sender === 'bot') {
+    console.log ('what do you want?')
     chat.innerHTML += `
       <section class="bot-msg">
-        <img src="assets/bot.png" alt="Bot" />
+        <img src="assets/operator.png" alt="Bot" />
         <div class="bubble bot-bubble">
           <p>${message}</p>
         </div>
       </section>
     `
   }
+
+  //Function that triggers click sound
+  messageAudio.currentTime = 0
+  messageAudio.play()
+
   // This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box
   chat.scrollTop = chat.scrollHeight
 }
 
-// Starts here
+//Startup greeting after load (timeout 1000)
 const greeting = () => {
-  showMessage(`Hello there, What's your name?`, 'bot')
-  // Just to check it out, change 'bot' to 'user' here 👆
+  showMessage(`Good day Sir/Ms, please type your name`, 'bot')
 }
 
-// Set up your eventlisteners here
+// First function handle name input
+const handleNameInput =  (event) => {
+  event.preventDefault()
+  const name = inputText.value
+  showMessage(`My name is ${name}`, 'user')
+  inputText.value = ''
+  setTimeout(() => destination(), 1000)
+}
+
+//Second function choose of destination
+const destination = () => {
```

Comment:
Your structure of the code is easy to follow in general. A small thing I noticed is that the code block between the curly bracket could be organized so it follows the same structure throughout the entire document. For example in this case I see the code block start on a new line after the first curly bracket (row 50 and 59).  On row 68, 76, 84 they start in the same line. Maybe it doesn't look like this in the actual VS code and is rather an automatic feature of this website. Again this is just a minor thing that could be fixed but can also be disregarded.